docs/source/gradio/edge_filter_gradio.html

```diff
@@ -0,0 +1,3 @@
+<script type="module" src="https://gradio.s3-us-west-2.amazonaws.com/3.2/gradio.js">
+</script>
+<gradio-app space="mipra/kornia-edge-detection"></gradio-app>
```

Comment:
```suggestion
<gradio-app space="kornia/kornia-edge-detection"></gradio-app>
```

This should change since the space has been transferred, otherwise the component won't load.

Reply:
thanks, changed the space name.